yt_astro_analysis/halo_analysis/halo_callbacks.py

```diff
@@ -36,6 +34,8 @@
 from yt.visualization.profile_plotter import \
     PhasePlot
 
+from more_itertools import always_iterable
```

Comment:
note for @brittonsmith : more_itertools should already be a transitive dependency of the project since it is the case of yt itself, but as of the latest developments in yt, it is imposed as a direct dependency there so this should be safe.